demisto_sdk/commands/common/tools.py

```diff
@@ -849,7 +849,8 @@ def get_entity_id_by_entity_type(data: dict, content_entity: str):
         if content_entity in (INTEGRATIONS_DIR, SCRIPTS_DIR):
             return data.get("commonfields", {}).get("id", "")
         elif content_entity == LAYOUTS_DIR:
```

Comment:
is the content entity the _directory_, or do we just reuse the constant with the same value?
If it's the latter, please find/create another constant

Reply:
its the directory